networkx/algorithms/centrality/group.py

```diff
@@ -84,6 +84,12 @@ def group_betweenness_centrality(G, C, normalized=True, weight=None, endpoints=F
     Zero edge weights can produce an infinite number of equal length
     paths between pairs of nodes.
 
+    The total number of paths between source and target is counted
+    differently for directed and undirected graphs. Directed paths
+    from "u" to "v" are counted as two directed paths and undirected
+    paths are counted as one paths.
```

Comment:
The text "paths from u to v" implies one direction of the path which is counted as one path, not two.
Tricky language here...  How about:
```suggestion
    The total number of paths between source and target is counted
    differently for directed and undirected graphs. Directed paths
    between "u" and "v" are counted as two possible paths (one each
    direction) while undirected paths between "u" and "v" are counted
    as one path. Said another way, the sum in the expression above is
    over all `s != t` for directed graphs and for `s < t` for undirected graphs.
```

Reply:
That reads well to me.